prime-router/metadata/fhir_mapping/hl7/resource/ServiceRequest.yml

```diff
@@ -313,18 +322,6 @@ extension_7:
       valueOf: OBR.20
       expressionType: HL7Spec
 
-extension_8:
```

Comment:
@victor-chaparro removing this extension breaks the integration tests. I can easily replace the data in OBR.21 is lost.

Reply:
Wrong extension removed, will replace.